src/modules/launcher/PowerLauncher/SettingsWatcher.cs

```diff
@@ -56,14 +56,14 @@ public void OverloadSettings()
                         shellSettings.UpdateSettings(overloadSettings);
                     }
 
-                    if (_settings.MaxResultsToShow != overloadSettings.properties.maximum_number_of_results)
+                    if (_settings.MaxResultsToShow != overloadSettings.Properties.maximum_number_of_results)

```

Comment:
maximum_number_of_results can be renamed to match Pascal Case

Reply:
Haven't gotten that far yet.  This small change hits 30 files already.